CONTRIBUTING.md

```diff
@@ -84,6 +84,20 @@ Go ahead and push your changes to GitHub using this command.
 git push
 ```
 
+## Cherry-pick PRs to release branches
+
+For example, to cherry-pick a merged PR to release-1.7 branch, on the local release-1.7 branch created off upstream/release-1.7, run:
+
+```sh
+git pull upstream release-1.7
+
+git checkout -b cherry-pick-pr
+
+git cherry-pick <commit-hash> -s
+
+git push --set-upstream origin cherry-pick-pr
+```
```

Comment:
Once cherry-pick the commit, the author will need to open a PR merging to the release branch, `release-1.7` for example.

Reply:
Done.